client/public/index.html

```diff
@@ -71,7 +67,7 @@
 
       Note that we insert the GTM script tag from JS to ensure that
       GTM only initializes itself once we've set the title tag.
-      For more details, see GoogleTagManager.tsx in this repository.
+      For more details, see Page.tsx in this repository.
     -->
     <meta name="jf-gtm-id" content="UA-67069242-5">
```

Comment:
Maybe we should move this and the HTML comment above it to the other GTM script tag above, so they're all in the same place?  Also, shouldn't the `content` attribute here be set to `GTM-NMPT5JP`?  Otherwise that bit of information is being completely lost, since the script tag above isn't actually using it...

Reply:
Yes good point! Ok I'll set that. Wait, now that I look at it more closely, the actual URL we use to access the GTM code is different here than it is on our other sites. Org Site and EFNYC make a request to `https://www.googletagmanager.com/gtm.js?id=XXX-XXXXX`
when the current configuration on WOW, the part that renders after the title tag renders, makes a request to `https://www.googletagmanager.com/gtag/js?id=XXX-XXXXX`. Any ideas why that is?

Reply:
I just did a quick diff between the code at both end points and they seem slightly different from each other. I wonder if one of the two is an older version of the GTM code? Hmmmm

Reply:
Ok so it looks like I should just use the more recently introduced URL structure for the GTM code, the one that WholeWhale provided for us.